go.mod

```diff
@@ -61,6 +61,7 @@ require (
 	github.com/rootless-containers/rootlesskit/v2 v2.3.1
 	github.com/spf13/cobra v1.8.1
 	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.9.0
```

Comment:
See note above. I am not sure why we would want to add testify.

Reply:
will remove it.